backends/exllamav3/model.py

```diff
@@ -450,9 +455,11 @@ def load_model_sync(self, progress_callback=None):
                     yield value
 
         for value in self.model.load_gen(
+            tensor_p=self.use_tp,
             reserve_per_device=self.autosplit_reserve,
             use_per_device=self.gpu_split,
             callback=progress_callback,
+            progressbar=True,
```

Comment:
iirc, this arg is for exl's progressbar. Since tabbyAPI uses its own, best to remove this.